### PR TITLE
 Pass events loop to glwindow test webvr implementation 

### PR DESCRIPTION
**Pull request by asajeffrey**
Pass the event loop to the glwindow webvr test device, so it can implement keyboard controls. The matching rust-webvr PR is https://github.com/servo/rust-webvr/pull/84

---

- [X] `./mach build -d` does not report any errors
- [X] `./mach test-tidy` does not report any errors
- [X] These changes do not require tests because this enables testing webvr pose

---
This change is [Reviewable](https://reviewable.io/reviews/servo/servo/23575)

### CODE REVIEW

**Comment by asajeffrey** [comment]
Depends on https://github.com/servo/rust-webvr/pull/84 and https://github.com/servo/servo/pull/23574

**Comment by asajeffrey** [comment]
Rebased.

File: Cargo.toml [hunk 1]
```diff
@@ -15,6 +15,8 @@ opt-level = 3
 # lto = false
 
 [patch.crates-io]
+rust-webvr = { git = "https://github.com/asajeffrey/rust-webvr.git", branch = "glwindow-keyboard-controls" }
```
Comment (Manishearth) [inline Cargo.toml]: this should be removed, yes?

**Review by Manishearth — APPROVED** [review]

**Comment by asajeffrey** [comment]
Rather annoyingly, this depends on #23574 which is in limbo right now, due to glutin's requirements around resize events. Sigh.

File: Cargo.toml (hunk 1 above)
Comment (asajeffrey) [reply Cargo.toml]: Done.

**Pull request merged**